source/javascripts/components/SplashModal/SplashModal.scss

```diff
@@ -0,0 +1,13 @@
+.ModalHeader {
+	height: 200px;
+	background: url("https://www.warrenphotographic.co.uk/photography/rubs/03656.jpg");
+	background-position: center;
+	background-size: cover;
+	border-radius: inherit;
+	border-bottom-left-radius: 0;
+	border-bottom-right-radius: 0;
+
+	a.Base--clickable.Link {
+		color: #fff;
```

Comment:
Do it with a constant, put it in _constants.scss as `$def-white`.

Reply:
I'll use the BitKit globals instead `var(--color-white)`